elasticsearch/fetch-artifacts-koji.yaml

```diff
@@ -1,3 +1,4 @@
 - nvr: org.elasticsearch-elasticsearch-6.8.1.redhat_00006-1
 - nvr: com.amazon.opendistroforelasticsearch-opendistro_security-0.10.0.4_redhat_00001-1
 - nvr: org.elasticsearch.plugin.prometheus-prometheus-exporter-6.8.1.0_redhat_00001-1
+- nvr: org.elasticsearch.plugin.ingest.openshift-ingest-plugin-6.8.1.0_redhat_00001-1
```

Comment:
This will break cpaas because it wont be found

Reply:
I will comment it out for now... but once the PNC build it setup and running I can put it back, right?